### PR TITLE
python scripts for sign and uploading jars to staging repo

### PR DESCRIPTION
**Pull request by ZhijunZhao**
@yungez @xscript @aovechki @yaweiw

### CODE REVIEW

File: .release-automation/run.py [hunk 1]
```diff
@@ -0,0 +1,21 @@
+"""This module releases local built jars to staging repo."""
+
+import json
+import utilities
+
+def main():
+    """Release jars to staging repo"""
```
Comment (xscript) [inline .release-automation/run.py]: Use separate lines for quotes to increase readability

File: .release-automation/config.json [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    "moduleNames": [
+        "azure-spring-boot-parent",
+        "azure-spring-boot-bom",
+        "azure-support",
+        "azure-ad-integration-spring-boot-starter",
+        "azure-documentdb-spring-boot-starter",
+        "azure-storage-spring-boot-starter",
+        "azure-keyvault-secrets-spring-boot-starter"
+    ],
+    "srcFolder": "C:/Users/zhijzhao/.m2/repository/com/microsoft/azure",
```
Comment (yungez) [inline .release-automation/config.json]: is it ok to use your personal folder name here?

File: .release-automation/utilities.py [hunk 3]
```diff
@@ -0,0 +1,215 @@
...
+import subprocess
+import time
+import requests
+from requests.auth import HTTPBasicAuth
+import xmltodict
+from jenkinsapi.jenkins import Jenkins
+
+def upload_jars(configs):
+    """Uploading local built jars to sign server"""
+    print("Upload jars to signing server...")
```
Comment (xscript) [inline .release-automation/utilities.py]: Use "Step m of n" to better show progress?

File: .release-automation/utilities.py [hunk 4]
```diff
@@ -0,0 +1,215 @@
...
+        subprocess.call(gpg_str)
+
+
+def generate_checksum(artifact_folder):
+    """Generate md5 and sh1 for all jars and poms"""
+    print("Generating checksum files...")
+    types = (os.path.join(artifact_folder, "*.jar"), os.path.join(artifact_folder, "*.pom"))
+    files_grabed = []
+    for file in types:
+        files_grabed.extend(glob.glob(file))
```
Comment (xscript) [inline .release-automation/utilities.py]: `files_grabed = [glob.glob(file) for file in types]`

**Pull request merged**